Dockerfile

```diff
@@ -8,6 +8,8 @@ RUN apt-get update -qq && apt-get install -y \
 ADD . /app
 WORKDIR /app
 
+RUN git rev-parse HEAD > COMMIT_HASH.txt && rm -rf .git
```

Comment:
I don't know if this will work as `.git` is in the `.dockerignore` file - perhaps it's better to just pass the env var into the Dockerfile as an `ARG` using circleci's `$CIRCLE_SHA1`

Reply:
It’s removed from the `.dockerignore` above https://github.com/artsy/force/pull/2851/files#diff-f7c5b4068637e2def526f9bbc7200c4eL1

Reply:
👀 ooo boy haha - okay well then it works, but maybe it's better to make it a build arg rather than copying in the entire repo

Reply:
I'll fix the copying `.git` thing. Then we should be ready to test on staging?